x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -202,12 +190,6 @@ export const hasTimestampFields = async (args: {
   return { foundNoIndices: false, warningMessage: undefined };
 };
 
-export const checkPrivileges = async (
```

Comment:
unused, we use `checkPrivilegesFromEsClient` directly in the rule execution logic